bin/tests/test_csv.py

```diff
@@ -31,23 +31,35 @@ def _test_random_splitter(self, expected_splits):
             col = self.csv_processing.data['split:split:int'][i]
             self.assertEqual(self.csv_processing.data['split:split:int'][i], expected_splits[i])
 
-    def _add_noise(self):
-        self.csv_processing.add_noise(self.configs['noise'])
+    def _transform(self):
+        self.csv_processing.transform(self.configs['transform'])
 
-    def _test_value_from_column(self, column_name, expected_value, position=0):
+    def _test_first_value_from_column(self, column_name, expected_value, position=0):
         """
         It tests the first value of a specific column.
         """
         observed_value = list(self.csv_processing.data[column_name])[position]
         if isinstance(observed_value, float):
             observed_value = round(observed_value, 2)
         self.assertEqual(observed_value, expected_value)
+    
+    def _test_all_values_in_column(self, column_name, expected_values):
+        """
+        It tests all the values of a specific column.
+        """
+        observed_values = list(self.csv_processing.data[column_name])
+        for i in range(len(observed_values)):
+            if isinstance(observed_values[i], float):
+                observed_values[i] = round(observed_values[i], 2)
```

Comment:
increase the rounding to 6 or 8 afterall floats hold 16 points digits 2 seems very small.

Reply:
done